pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/proc_delete_modules.adoc

```diff
@@ -0,0 +1,17 @@
+[id="delete_modules_{context}"]
+
+= Deleteing a module in {ProductShortName}
+
+Deleting a module in {ProductShortName}.
+
+.Prerequisites
+
+//* You are assigned a publisher or admin role in {ProductShortName}.
+* You are logged in.
+
+.Procedure
+
+. Navigate to the link:{LinkToSearchPage}[Search page].
+. Select the check box of the module you want to publish.
+. Click *Delete*.
+. In the confirmation screen, click *OK* to confirm.
```

Comment:
It'd be nice to have a conclusion after the procedure, such as "The deleted module is no longer available in {ProductShortName}, or some such. 

Also, I'm curious to know whether the module simply gets unlisted from Pantheon or does it get deleted from the Git repository, This could be out of scope for the moment though.

Reply:
done